README.md

```diff
@@ -1,46 +1,41 @@
 rust-GSL [![Build Status](https://api.travis-ci.org/GuillaumeGomez/rust-GSL.png?branch=master)](https://travis-ci.org/GuillaumeGomez/rust-GSL) [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/GuillaumeGomez/rust-GSL?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 ========
 
-A __Rust__ binding for the [GSL library](http://www.gnu.org/software/gsl/) (the GNU Scientific Library).
+A __Rust__ binding for the [GSL library] (the GNU Scientific Library).
 
 ##Installation
 
-This binding requires the [GSL library](http://www.gnu.org/software/gsl/) library to be installed.
+This binding requires the [GSL library] library to be installed.
 
-To build it, please use :
+This crate works with Cargo and is on [crates.io].  Just add the
+following to your `Cargo.toml` file:
 
-```Shell
-> make
-```
-
-This command builds __rgsl__, the examples and the documentation.
-
-You can build them separatly too.
-
-```Shell
-> make rgsl
-> make examples
-> make doc
+```toml
+[dependencies]
+GSL = "*"
 ```
 
-Since this project supports cargo, you can also build it like this :
+Add the following line to your source code:
 
-```Shell
-> cargo build
+```rust
+extern crate rgsl;
 ```
 
 ##Documentation
 
-You can access the __rgsl__ documentation locally, just build it :
+You can access the __rgsl__ documentation locally, just build it:
 
 ```Shell
-> make doc
+> cargo doc --open
 ```
 
-Then open this file with an internet browser :
-file:///{rgsl_location}/doc/rgsl/index.html
+Then open this file with an internet browser:
+`file:///{rgsl_location}/target/doc/rgsl/index.html`
 
 You can also access the latest build of the documentation via the internet [here](http://rust-ci.org/GuillaumeGomez/rust-GSL/doc/rgsl/).
 
 ## License
 __rust-GSL__ is a wrapper for __GSL__, therefore inherits the [GPL license](http://www.gnu.org/copyleft/gpl.html).
+
+[crates.io]: https://crates.io/crates/GSL
```

Comment:
I'm not sure to understand what you wanted to do here. In addition, it seems it doesn't render in github. Do you wanted to do something else ?